esmvalcore/_config.py

```diff
@@ -172,13 +172,22 @@ def get_project_config(project):
 
 
 def get_institutes(variable):
-    """Return the institutes given the dataset name in CMIP5."""
+    """Return the institutes given the dataset name in CMIP5 and CMIP6."""
     dataset = variable['dataset']
     project = variable['project']
     logger.debug("Retrieving institutes for dataset %s", dataset)
+    if project in 'CMIP6':
```

Comment:
```suggestion
    if CMOR_TABLES[project].institutes:
```
would work better if we add more cmor tables later on